website/packages/docs/src/pages/pkg-webpack-loader.mdx

```diff
@@ -307,3 +307,18 @@ See [Shorthand properties](/shorthand) for more information.
 
 - Type: `boolean`
 - Default: `false`
+
+This should be set on both the `@compiled/webpack-loader`:
+
+```js
+{
+  loader: '@compiled/webpack-loader',
+  options: { sortShorthand: true }
```

Comment:
Is this even right?  It's a bit unclear to me…

Reply:
docs changes look reasonable, let me double check in the repo

Reply:
oh this is incorrect, `sortShorthand` exists for `CompiledExtractPlugin` but not `@compiled/webpack-loader`

see current docs https://compiledcssinjs.com/docs/pkg-webpack-loader where `sortShorthand` is listed under the options for `CompiledExtractPlugin` but not listed under the options for `@compiled/webpack-loader`

Reply:
Oh, I got how this documentation works now 😅   I'll just keep the example so someone doesn't misunderstand it like I did.